src/services/behaviors/index.ts

```diff
@@ -138,3 +138,5 @@ export const trigger = (
     });
   });
 };
+
+export { createActionHandler, triggerBehaviors } from './helpers';
```

Comment:
was there a particular intention in putting these files under helpers instead of into this file directly?

Reply:
It's not directly related to the purpose of the service. Can move if you think that makes more sense.